pom.xml

```diff
@@ -316,7 +316,7 @@
       <dependency>
         <groupId>io.netty</groupId>
         <artifactId>netty-all</artifactId>
-        <version>4.0.29.Final</version>
+        <version>4.0.53.Final</version>
```

Comment:
can you javadoc here we need this version or later to ensure shading works on netty?

Reply:
Updated, thanks.